sources/assets/shells/aliases.d/fzf

```diff
@@ -1,4 +1,5 @@
 alias fzf-wordlists='find /opt/rockyou.txt /opt/seclists /usr/share/wordlists /usr/share/wfuzz /usr/share/dirb -type f | fzf'
+alias fzf-rules='find /usr/share/hashcat/rules /usr/share/hashcat/rules/hybrid -type f |fzf'
```

Comment:
`/usr/share/hashcat/rules/hybrid` is not needed since the parent folder is already in the command
and please add a space between the pipe (`|`) and `fzf`

You'll also need to add `/opt/rules/` in the alias as explained in another comment